examples/hooks-ssr/webpack.config.js

```diff
@@ -0,0 +1,63 @@
+const path = require('path');
```

Comment:
Why do you need a custom Webpack config? Can't a standard Parcel setup do the job?

Reply:
I think that's because it's copied from the React InstantSearch v6 server side rendering demo, which indeed doesn't require a custom webpack setup, it just has that because of latency and fits more use cases

Reply:
I used the same config as the previous SSR example to compare them more easily.

Reply:
Alright, we can probably simplify it when we deprecate RIS DOM.